<?xml version="1.0"?>

<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
	<param name="base_global_planner" value="carrot_planner/CarrotPlanner" />
	<param name="base_global_planner" value="global_planner/GlobalPlanner" />
    <rosparam file="$(find exp_final)/param/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find exp_final)/param/costmap_common_params.yaml" command="load" ns="local_costmap"/>
    <rosparam file="$(find exp_final)/param/local_costmap_params.yaml" command="load"/>
    <rosparam file="$(find exp_final)/param/global_costmap_params.yaml" command="load"/>
    <rosparam file="$(find exp_final)/param/base_local_planner_params.yaml" command="load"/>
    <rosparam file="$(find exp_final)/param/move_base_params.yaml" command="load"/>
  </node>
</launch>